decidim-initiatives/spec/controllers/decidim/initiatives/admin/committee_requests_controller_spec.rb

```diff
@@ -11,7 +11,7 @@ module Admin
         let(:organization) { create(:organization) }
         let(:initiative) { create(:initiative, :created, organization:) }
         let(:admin_user) { create(:user, :admin, :confirmed, organization:) }
-        let(:user) { create(:user, :confirmed, organization:) }
+        let(:user) { create(:user, :confirmed, admin_terms_accepted_at: Time.current, organization:) }
```

Comment:
You can also use the `:admin_terms_accepted` trait here.
```suggestion
        let(:user) { create(:user, :confirmed, :admin_terms_accepted, organization:) }
```